adam-core/src/main/scala/org/bdgenomics/adam/rdd/contig/NucleotideContigFragmentRDD.scala

```diff
@@ -169,7 +170,7 @@ case class NucleotideContigFragmentRDD(
    * @param region Reference region over which to get sequence.
    * @return String of bases corresponding to reference sequence.
    */
-  def getReferenceString(region: ReferenceRegion): String = {
+  def extract(region: ReferenceRegion): String = {
```

Comment:
I think it's fine for now, with an eye to refactor if/when the Slice stuff is added, between versions 0.20 and 0.21 presumably